changelog/7464.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added support for ``NO_COLOR`` and ``FORCE_COLOR`` environment variables to control colored output.
+
+For more information, see `the docs <https://docs.pytest.org/en/stable/reference.html#environment-variables>`__.
```

Comment:
This could/should use an internal reference (Python uses `using-on-envvars`).
In general (internal) documentation links should not go through external resources/protocols.

Reply:
True, that escaped my mind, thanks. 👍

Reply:
Done: https://github.com/pytest-dev/pytest/pull/7474

Thanks again